extensions/git/src/util.ts

```diff
@@ -149,6 +149,16 @@ export async function mkdirp(path: string, mode?: number): Promise<boolean> {
 	return true;
 }
 
+export async function createOrAppendFile(path: string, data: string, options?: any): Promise<boolean> {
```

Comment:
I'd rather we go through the `applyEdit` vscode API, instead of raw node file editing.